pgdb.py

```diff
@@ -1662,10 +1662,9 @@ def connect(dsn=None,
             value = str(value)
             if not value or ' ' in value:
                 value = "'%s'" % (value.replace(
-                    "'", "\\'").replace('\\', '\\\\'),)
+                    '\\', '\\\\').replace("'", "\\\'"))
             dbname.append('%s=%s' % (kw, value))
```

Comment:
The second replace should have only two backslashes, though it will work since Python simply ignores the superfluous backslash (`"\'" == "'"`). The actual problem that you have fixed is that the two replacements were in the wrong order.

Reply:
I did not realize this. Thanks for noticing this. I've ammended and re-pushed.